data/assets/05_dash-graph.css

```diff
@@ -0,0 +1,40 @@
+@import './01.css';
+
+/* Graphs in matrix view and chart view */
```

Comment:
(nit) There is not "graph" per se in the matrix view. What do those attributes control? Some kind of an animation?

Reply:
Good point, they don't seem to affect anything, unfortunately I can't ask the UI person as he is no longer available.
Removing